.github/workflows/guide-website-update.yml

```diff
@@ -1,36 +0,0 @@
-name: Guide Web Site Deploy
```

Comment:
We need this one to auto update changes.

Reply:
I added it.